dao/src/test/java/com/carhouse/dao/config/TestConfiguration.java

```diff
@@ -12,4 +13,27 @@
 @Configuration
 public class TestConfiguration {
 
+    @Bean
+    public ObjectMapper getObjectMapper() {
+        return new ObjectMapper();
+    }
+
+    @Profile("h2-database")
+    @Bean
+    public PropertySourcesPlaceholderConfigurer propertySourcesPlaceholderConfigurerH2() {
+        PropertySourcesPlaceholderConfigurer properties = new PropertySourcesPlaceholderConfigurer();
+        properties.setLocation(new ClassPathResource("test-h2-jdbc-connection.properties"));
+        properties.setIgnoreResourceNotFound(false);
+        return properties;
+    }
+
+    @Profile("mysql-database")
```

Comment:
Propose to investigate how to configure beans that by default it will be H@, but with some kind of parameters MySQL connection. It helps to remove a lot of annotations